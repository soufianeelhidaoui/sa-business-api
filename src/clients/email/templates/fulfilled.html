<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
    <style>
    </style>
  </head>
  <body>
    <div style="max-width: 600px; margin: 0 auto;">
    {{> headBand }}
      <div style= 'font-size: 16px;
        font-style: normal;
        font-weight: 400;
        line-height: 24px;
        font-family: VW Text, Arial;
        margin-bottom: 32px
        '>
        <h2>
          <center>Votre commande est disponible !</center>
        </h2>
        <p>Bonjour {{ orderDetails.customer.firstName }},</p>
        <p>Votre commande est prête à être retirée auprès de votre partenaire agréé. Pensez à vérifier les jours et heures d'ouverture de votre partenaire agréé.
        </p>
      </div>
      <hr style="color: #DFE4E8" />
      {{> dealerMap }}
      <div style="clear: both;" />
      {{> footer }}
    </div>
  </body>
</html>
