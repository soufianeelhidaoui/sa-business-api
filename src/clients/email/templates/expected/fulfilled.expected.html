<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
    <style>
    </style>
  </head>
  <body>
    <div style="max-width: 600px; margin: 0 auto;">
    <center>
      <img
        src="SKODA_API_URL/assets/shop/img/email/email-logo-skoda.png"
        alt="hero"
      />
    </center>
    <img
      src="SKODA_API_URL/assets/shop/img/email/email-skoda.jpg"
      alt="hero"
      style="width:100%"
    />
      <div style= 'font-size: 16px;
        font-style: normal;
        font-weight: 400;
        line-height: 24px;
        font-family: VW Text, Arial;
        margin-bottom: 32px
        '>
        <h2>
          <center>Votre commande est disponible !</center>
        </h2>
        <p>Bonjour Test,</p>
        <p>Votre commande est prête à être retirée auprès de votre partenaire agréé. Pensez à vérifier les jours et heures d'ouverture de votre partenaire agréé.
        </p>
      </div>
      <hr style="color: #DFE4E8" />
      
      <div style="font-family:VW Head, Arial;font-size:14px;font-weight:300;line-height:24px;text-align:left;color:#000;margin:0px auto;max-width:600px">
        <center>
        <h3 style="font-size:24px;font-weight:700;padding:0 50px">Votre concessionnaire</h3>
        </center>
      </div>
      
      <div style="float: left;">
        <h6 style="font-size:18px;font-style: normal;line-height: 24px; font-family:VW Head, Arial">SAS ESPACE 3000 MONTBELIARD</h6>
        <p style="font-size:14px;font-style: normal;line-height: 18px; font-family:VW Head, Arial">
          <img src="SKODA_API_URL/assets/shop/img/icon-pin.png" width="15px" height="15px"/>
          12 Avenue de l&#x27;Europe
        </p>
        <p style="font-size:14px;font-style: normal;line-height: 18px; font-family:VW Head, Arial">
          <img src="SKODA_API_URL/assets/shop/img/icon-phone.png" width="15px" height="15px"/>
          0381364290
        </p>
        <p>
          <a href="https://www.google.com/maps/dir//+12 Avenue de l&#x27;Europe+25400+Arbouans" style="padding:5px 20px;border:1px solid;border-radius:25px;color:#000;text-decoration:none" target="_blank">
            <img src="SKODA_API_URL/assets/shop/img/icon-navigate.png" style="border:0;display:inline-block;outline:none;text-decoration:none;margin-right:5px" width="16" height="16" class="CToWUd" data-bit="iit">
            Voir l’itinéraire
        </a>
        </p>
      </div>
      
      <div style="float: right;">
        <div class="mj-column-per-43 mj-outlook-group-fix" style="font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;">
          <table border="0" cellpadding="0" cellspacing="0" role="presentation" width="100%">
              <tbody>
                  <tr>
                      <td style="vertical-align:top;padding:0;">
                          <table border="0" cellpadding="0" cellspacing="0" role="presentation" style width="100%">
                              <tbody>
                                  <tr>
                                      <td align="center" style="font-size:0px;padding:0;word-break:break-word;">
                                          <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="border-collapse:collapse;border-spacing:0px;">
                                              <tbody>
                                                  <tr>
                                                      <td style="width:214px;">
                                                          <img src="https://maps.googleapis.com/maps/api/staticmap?center='12 Avenue de l&#x27;Europe'+'25400'+'Arbouans'&zoom=14&size=214x214&maptype=roadmap&key=AIzaSyAwFtKQ_9R8khClNMXQwTh_sDMslMFEvs0" width="214" height="214"/>
                                                      </td>
                                                  </tr>
                                              </tbody>
                                          </table>
                                      </td>
                                  </tr>
                              </tbody>
                          </table>
                      </td>
                  </tr>
              </tbody>
          </table>
        </div>
      </div>
      <div style="clear: both;" />
      <div>
          <hr style="color: #DFE4E8" />
          <!--[if mso | IE]><table align="center" border="0" cellpadding="0" cellspacing="0" class="" style="width:600px;" width="600" ><tr><td style="line-height:0px;font-size:0px;mso-line-height-rule:exactly;"><![endif]-->
          <!--[if mso | IE]><table align="center" border="0" cellpadding="0" cellspacing="0" class="wrapper-outlook" style="width:600px;" width="600" ><tr><td style="line-height:0px;font-size:0px;mso-line-height-rule:exactly;"><![endif]-->
          <div class="wrapper" style="margin:0px auto;">
              <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" style="width:100%;">
                  <tbody>
                      <tr>
                          <td style="direction:ltr;font-size:0px;text-align:center;">
                              <!--[if mso | IE]><table role="presentation" border="0" cellpadding="0" cellspacing="0"><tr><td class="" style="vertical-align:top;width:520px;" ><![endif]-->
                              <div class="mj-column-per-100 mj-outlook-group-fix" style="font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;">
                                  <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="vertical-align:top;" width="100%">
                                      <tbody>
                                          <tr>
                                              <td align="left" style="font-size:0px;padding:10px;word-break:break-word;">
                                                  <div style="font-family:SKODANext, Arial;font-size:12px;font-weight:300;line-height:24px;text-align:left;color:#878787;">
                                                      <p>SKODA, division de Volkswagen Group France responsable du traitement, traite les données à caractère personnel recueillies afin de vous permettre de recevoir, via l'adresse email que vous avez renseignée, les coordonnées et les références de votre Réparateur Agréé préféré sur la base légale de notre intérêt légitime, dans le cadre de la gestion de notre relation client.</p>
                                                      <p>Pour en savoir plus sur la gestion de vos données personnelles et pour exercer vos droits, veuillez consulter notre <a href="https://www.skoda.fr/informations/politique-de-confidentialite?_gl&#x3D;1*1mqpfd6*_gcl_au*OTk2NjU4OTQ1LjE3MTc1MTUwMjk.">Politiques de confidentialité</a> </p>
                                                  </div>
                                              </td>
                                          </tr>
                                      </tbody>
                                  </table>
                              </div>
                          </td>
                      </tr>
                  </tbody>
              </table>
          </div>
        </div>
    </div>
  </body>
</html>
